packages/product-react/src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+
+import Radio from './Radio';
+
+describe('Radio', () => {
+  test('it renders a radio', () => {
```

Comment:
you might as well use `it` instead of `test` if you're going to start the strings with "it"

Reply:
I change the work thanks